lib/game.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal:true
+
+class Game
+    attr_accessor :player1, :player2
+    attr_reader :current_player, :board
+
+    def initialize
+        @board = Board.new
+        @player1 = Player.new('X')
+        @player2 = Player.new('O')
+        @current_player = [@player1,@player2].sample(1).first
+    end

```

Comment:
According to what I saw up there, you can initialize the Game with players as parameters